getting-started-config.yaml

```
@@ -46,14 +46,19 @@ resources:
     description: These guidelines will help you use the Vipps payments
     markdownUrl: vipps-design-guidelines/master/README.md
     githubUrl: https://github.com/vippsas/vipps-design-guidelines/blob/master/README.md
+  - title: GitHub resources
+    slug: github-resources
+    description: All developer resources on GitHub
+    markdownUrl: vipps-developers/blob/master/README.md
```

This file will not work correctly, because it has more than one H1 header.
However, if you would like to add, please remove the `blob` from the url, since it is using the raw view of the file, from the domain: https://raw.githubusercontent.com/
```suggestion
    markdownUrl: vipps-developers/master/README.md
```